packages/frontend-2/lib/projects/composables/team.ts

```diff
@@ -27,22 +52,17 @@ export function useTeamManagePermissionsInternals(params: {
   }
 }
 
-export function useTeamDialogInternals(params: {
-  props: {
-    project: Ref<ProjectPageTeamDialogFragment>
-  }
-}) {
-  const {
-    props: { project }
-  } = params
+export function useTeamInternals(
+  projectData: ComputedRef<ProjectPageTeamInternals_ProjectFragment | undefined>
+) {
+  const { isGuest: isServerGuest, activeUser } = useActiveUser()
 
-  const { isOwner, isServerGuest, activeUser } =
-    useTeamManagePermissionsInternals(params)
+  const isOwner = computed(() => projectData.value?.role === Roles.Stream.Owner)
```

Comment:
Why duplicate the same logic `useTeamManagePermissionsInternals` already did? You should add back the `useTeamManagePermissionsInternals` invocation so that the code is more DRY